zerver/models.py

```diff
@@ -609,6 +610,7 @@ class UserProfile(ModelReprMixin, AbstractBaseUser, PermissionsMixin):
     DEFAULT_UPLOADS_QUOTA = 1024*1024*1024
 
     quota = models.IntegerField(default=DEFAULT_UPLOADS_QUOTA) # type: int
+    timezone = models.CharField(max_length=50, default='UTC')  # type: Text
```

Comment:
Where does 50 come from?  Might want to look at lengths and put a comment for what the longest one is and how long it is...